lib/l10n_utils/management/commands/open_ftl_pr.py

```diff
@@ -0,0 +1,105 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from datetime import datetime
+from os import getenv
+from subprocess import CalledProcessError
+
+from django.core.management.base import CommandError
+from django.conf import settings
+from django.template.defaultfilters import slugify
+from django.utils.functional import cached_property
+
+from bedrock.utils import github
+
+from dirsync import sync
+
+from ._ftl_repo_base import FTLRepoCommand
+
+
+GIT_HASH = getenv('CI_COMMIT_SHORT_SHA', None)
+
+
+class Command(FTLRepoCommand):
+    help = "Open a pull-request on the L10n Team's reop for FTL file changes"
+
+    def handle(self, *args, **options):
+        super().handle(*args, **options)
+        self.update_l10n_team_files()
+        self.create_branch()
+        self.sync_dirs()
+        changed = self.commit_changes()
+        if changed:
+            self.push_changes()
+            self.open_pr()
+
+    @cached_property
+    def branch_name(self):
+        if GIT_HASH:
+            branch_suffix = GIT_HASH
+        else:
+            branch_suffix = slugify(datetime.now().isoformat())
+
+        return f'update-from-bedrock-{branch_suffix}'
+
+    @property
+    def commit_message(self):
+        message = 'Updates from bedrock\n\n'
+        if GIT_HASH:
+            message += f'From file changes in https://github.com/mozilla/bedrock/commit/{GIT_HASH}'
+        else:
+            message += 'From file changes in https://github.com/mozilla/bedrock/commits/master'
+
+        return message
+
+    def sync_dirs(self):
+        sync(settings.FLUENT_LOCAL_PATH, self.l10n_repo.path, 'sync', content=True)
+
+    def create_branch(self):
+        self.l10n_repo.git('checkout', '-b', self.branch_name)
+        self.stdout.write(f'Created branch: {self.branch_name}')
+
+    def commit_changes(self):
+        self.config_git()
+        self.l10n_repo.git('add', '.')
+
+        try:
+            self.l10n_repo.git('commit', '-m', self.commit_message)
+        except CalledProcessError:
+            self.stdout.write('No changes to commit')
+            return False
+
+        self.stdout.write('Committed changes to local repo')
+        return True
+
+    def push_changes(self):
+        try:
+            self.l10n_repo.git('push', self.git_push_url, 'HEAD')
+        except CalledProcessError:
+            raise CommandError(f'There was a problem pushing to {self.l10n_repo.remote_url}')
```

Comment:
If we reach this point, next run should not be affected b/c it will be a new branch, correct?